adds how to add pairing field in ado cards (#559)
microsoft/code-with-engineering-playbook: agile-development/collaboration/add-pairing-field-ado-cards.md

> # How to add a pairing custom field in ADO User Stories

Reviewer: ```suggestion
# How to add a pairing custom field in Azure DevOps User Stories
```

This document outlines the benefits of adding a custom field of type _Identity_ in [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/user-guide/what-is-azure-devops) (ADO) user stories, prerequisites, and a step-by-step guide.

## Benefits of adding a custom field

> Having the names of both individuals [pairing on a story](./readme.md) visible on the ADO cards can be helpful during sprint ceremonies and lead to greater accountability by the pairing assignee. For example, it is easier to keep track of the individuals assigned stories as part of a pair during sprint planning by using the "pairing names field". During stand-up it can also help the process lead filter stories assigned to the individual (both as an owner or as a pairing assignee) and show these on the board. Furthermore, the pairing field can provide an additional data point for reports and burndown rates.

Reviewer 2: Using Azure DevOps instead of ADO is recommended through out the markup

Author: Good idea! All ADO references now spell out Azure DevOps.

> Having the names of both individuals [pairing on a story](./readme.md) visible on the ADO cards can be helpful during sprint ceremonies and lead to greater accountability by the pairing assignee. For example, it is easier to keep track of the individuals assigned stories as part of a pair during sprint planning by using the "pairing names field". During stand-up it can also help the process lead filter stories assigned to the individual (both as an owner or as a pairing assignee) and show these on the board. Furthermore, the pairing field can provide an additional data point for reports and burndown rates.

Reviewer 2: ```suggestion
Having the names of both individuals [pairing on a story](./readme.md) visible on the ADO cards can be helpful during sprint ceremonies and lead to greater accountability by the pairing assignee. For example, it is easier to keep track of the individuals assigned stories as part of a pair during sprint planning by using the "pairing names" field. During stand-up it can also help the Process Lead filter stories assigned to the individual (both as an owner or as a pairing assignee) and show these on the board. Furthermore, the pairing field can provide an additional data point for reports and burndown rates.
```

## Prerequisites

Prior to customizing ADO, review [Configure and customize Azure Boards](https://docs.microsoft.com/en-us/azure/devops/boards/configure-customize).

> In order to add a custom field to user stories in ADO changes must be made on an Organizational setting. This document therefore assumes use of an existing Organization in ADO and that the user account used to make these changes is a member of the [Project Collection Administrators Group](https://docs.microsoft.com/en-us/azure/devops/organizations/security/set-project-collection-level-permissions).

Reviewer 3: Might be worth making "**Organizational setting**" bold so it stands out

> In order to add a custom field to user stories in ADO changes must be made on an **Organizational setting**. This document therefore assumes use of an existing Organization in ADO and that the user account used to make these changes is a member of the [Project Collection Administrators Group](https://docs.microsoft.com/en-us/azure/devops/organizations/security/set-project-collection-level-permissions).

Reviewer: ```suggestion
In order to add a custom field to user stories in ADO changes must be made as an **Organizational setting**. This document therefore assumes use of an existing Organization in ADO and that the user account used to make these changes is a member of the [Project Collection Administrators Group](https://docs.microsoft.com/en-us/azure/devops/organizations/security/set-project-collection-level-permissions).
```

## Step-by-step to change the organization settings

> ### Duplicate the process currently in use

Reviewer: I would suggest making this an ordered list instead of sub headings

Author: Good idea!

Navigate to the Organization Settings, within the Boards / Process tab.

![Organization Settings](./images/ado-organization-settings.png)

### Select the Process type, click on the icon with three dots ... and click 'Create inherited process'

> ![Create Interited Process](./images/ado-create-inherited-process.png)

Reviewer: ```suggestion
![Create Inherited Process](./images/ado-create-inherited-process.png)
```

### Click on the newly created inherited process

As you can see in the example below, we called it 'Pairing'

> ![Select Interited Process](./images/ado-pairing-process.png)

Reviewer: ```suggestion
![Select Inherited Process](./images/ado-pairing-process.png)
```

### Click on the work item type User Story

![Modify User Story](./images/ado-user-story-process.png)

### Click New Field

![Add the new field](./images/ado-new-field.png)

### Give it a Name and select Identity in Type

Click on Add Field.

![Add the new field](./images/ado-add-field-to-user-story.png)

This completes the change in Organization settings. The rest of the instructions must be completed under Project Settings.

> ## Step-by-step to change the project settings

Reviewer 3: Recommend editing to "Next, change the project settings" as it might otherwise look like two different sections of the article, ie two different ways of doing it with the step by step organizational and project settings sections.
